credentialdigger/client.py

```diff
@@ -1191,11 +1196,14 @@ def _analyze_discovery(d):
         if debug:
             model_name = model_manager.model.__class__.__name__
             logger.debug(f'Analyzing discoveries with model {model_name}')
-
+            total_discoveries = len(discoveries)
             false_positives = 0
-            for i in tqdm(range(len(discoveries))):
-                false_positives += _analyze_discovery(discoveries[i])
-
+            with Progress() as progress:
+                scanning_task = progress.add_task("Scanning discoveries...",
+                                                  total=total_discoveries)
+                for i in range(total_discoveries):
```

Comment:
again, we don't need the index anymore so we can loop directly on the discoveries